docs/deploy-and-review-process.md

```diff
@@ -0,0 +1,59 @@
+# Deploy and review process
+
+This documentation outlines our current processes for how we deploy the latest changes to ethereum.org, and how we prioritize items in our review process.
+
+## Deploy process
+
+Anything merged into the `dev` branch of [ethereum.org](http://ethereum.org) will go live on ethereum.org at the next release. Currently, we deploy the latest changes to [ethereum.org](http://ethereum.org) twice per week.
+
+These are currently:
+
+Monday between 17:00 and 23:00 UTC.
+
+Thursday between 17:00 and 23:00 UTC.
+
+Occasionally, we’ll also do a patch deploy to fix an urgent or widespread issue.
+
+## Review process
+
+### General review process
+
+We use a first-in, first-out system for reviewing pull requests. The longer a pull request has been open, the higher priority it is for our team to review. In some cases—for example, fixing a high-priority issue or merging low-hanging fruit for a deploy—we will stray from this process and use our best judgement to get higher-impact changes deployed more quickly.
+
+We aim to have every new PR merged or closed within 30 days of opening. As outlined in the following sections, different types of pull requests do have different levels of priority, and this may influence how promptly a pull request is acted on.
```

Comment:
Should the aim be to have PRs "merged" vs at least being discussed and iterated on? No hard opinion either way

Reply:
I think it's ok to say the aim here is that. If it goes over that time due to discussion or iteration being needed I think that's a case-by-case situation.